fab/fabfile.py

```diff
@@ -70,6 +70,7 @@
 RELEASE_RECORD = 'RELEASES.txt'
 env.linewise = True
 env.colorize_errors = True
+env['sudo_prefix'] += '-H '
```

Comment:
@dannyroberts was getting EACCESS errors when running bower install because it was trying to read `/home/brudolph/.config`. with this prefix, sudo uses the `cchq` HOME variable. i mention this because i think it should fix those git warnings you were seeing